### PR TITLE
Created media query mixins

### PR DESCRIPTION
**Pull request by sanlucca**
Created three media query mixins : 
1) media_smaller_than(breakpoint)
2) media_larger_than(breakpoint)
3) media_between(breakpoint1, breakpoint2)

They throw warnings if the breakpoint used isn't defined

### CODE REVIEW

File: packages/scss/src/mixins/_responsiveness.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@mixin media_smaller_than($breakpoint) {
+	@if map-has-key(map-get($theme,"breakpoints"), $breakpoint) {
```
Comment (lucienbertin) [inline packages/scss/src/mixins/_responsiveness.scss]: `_theme("brerakpoints")` is the same as `map-get($theme,"breakpoints")`

Comment (sanlucca) [inline packages/scss/src/mixins/_responsiveness.scss]: Yes, but if the key doesn't exist, it throws a general error (stating that the key $breakpoint doesn't exist) without specifying the fact that it is a breakpoint that doesn't exist (error handled by these mixins)

Comment (lucienbertin) [inline packages/scss/src/mixins/_responsiveness.scss]:
```
$stuff: _safeGet($theme, "breakpoints." + $breakpoint)
@if $stuff
  ...
```

Comment (lucienbertin) [inline packages/scss/src/mixins/_responsiveness.scss]: also i said that this
```
map-has-key(_theme("breakpoints"), $breakpoint)
```
is equivalent to that
```
map-has-key(map-get($theme,"breakpoints"), $breakpoint)
```

**Pull request merged**